direct/src/showbase/Loader.py

```diff
@@ -949,7 +949,7 @@ def loadMusic(self, *args, **kw):
             return None
 
     def loadSound(self, manager, soundPath, positional = False,
-                  callback = None, extraArgs = []):
+                  callback = None, okMissing=None, extraArgs = []):
```

Comment:
Style consistency nitpick
Also, this will break if people passed in the arguments positionally, rather than by keyword, so let's preserve the order

Reply:
Fixed the argument order in https://github.com/panda3d/panda3d/pull/1443/commits/3ef954c80be2992f4a0eaa9b9a4e50393583be53